eng/common/instructions/azsdk-tools/validate-package.instructions.md

```diff
@@ -0,0 +1,12 @@
+mode: 'agent'
+tools: ['azsdk_package_run_check']
+---
+
+## Goal:
+Validate generated and custom code in local SDK repositories. Assist users in fixing any issues found.
+
+## Step 1: Run Validation
+Use `azsdk_package_run_check` to validate the package. This tool checks for common issues in the package. To fix issues found, pass in the appropriate argument to the command.
```

Comment:
does the tool currently work for all languages and both management plane and data plane? If not, is that a blocker for merging this PR?